karma.conf.js

```diff
@@ -0,0 +1,73 @@
+/*global module*/
+'use strict';
+
+module.exports = function(config) {
+	config.set({
+
+		// base path that will be used to resolve all patterns (eg. files, exclude)
+		basePath: '',
+
+
+		// frameworks to use
+		// available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+		frameworks: ['mocha', 'browserify'],
+
+
+		// list of files / patterns to load in the browser
+		files: [
+			'http://polyfill.webservices.ft.com/v1/polyfill.js?ua=safari/4',
```

Comment:
tbh, all the comments here are generated automatically, and that's the only line I've added. I haven't added that comment in the tests elsewhere, but might be a good idea to start adding it